lib/vagrant-libvirt/action/forward_ports.rb

```diff
@@ -72,7 +72,7 @@ def compile_forwarded_ports(config)
           config.vm.networks.each do |type, options|
             next if options[:disabled]
 
-            next unless type == :forwarded_port && options[:id] != 'ssh'
+            next unless type == :forwarded_port && !@env[:machine].provider_config.forward_ssh_port && options[:id] != 'ssh'
```

Comment:
Looking at the very original patch for the forwarded ports I think the expectation was the skipping forwarding the default ssh port should be removed at some point
https://github.com/vagrant-libvirt/vagrant-libvirt/pull/139/commits/bb5d0817894b54f8ecabe1fb9a8496f13b4adaf8#diff-a04ee612a672226ed5bdbe2817ec5bb9de835ca845afe54aaa4730473b3250eeR63

So I think best to invert this config option and make it possible to use to disable forwarding this if it causes problems, but otherwise allow the default forwarding of the ssh port on 2022 to exist.